### PR TITLE
[#69] Created Jetpack compose navigation sample

### LINKED ISSUE #69: Add sample for jetpack compose navigation
Create a sample project which demonstrates how someone could add Appyx to an existing Jetpack Compose Navigation project.

### PR DESCRIPTION
**Pull request by LachlanMcKee**
## Description
Fixes #69

## Check list

- [x] I have updated `CHANGELOG.md` if required.
- [x] I have updated documentation if required.

### CODE REVIEW

File: documentation/how-to-use-appyx/sample-apps.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ The `:app` module showcases **Appyx** itself with multiple levels of navigation,
 
 ## Appyx + Jetpack Compose Navigation example 
 
-Coming soon!
+The `:samples:navigation-compose` module demonstrates how to use Appyx within Google's Jetpack Compose Navigation library.
+This example may be useful if you need to migrate to Appyx gradually.
+
+1. Go to the [GitHub project](https://github.com/bumble-tech/appyx)
```
Comment (LachlanMcKee) [inline documentation/how-to-use-appyx/sample-apps.md:18]: Perhaps we want to reduce the repetition?

Comment (zsoltk) [reply documentation/how-to-use-appyx/sample-apps.md:18]: Sure. You could pull out a section to the top "## Where to find the sample apps"

File: samples/navigation-compose/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Appyx sample - Navigation Compose
+
+This sample demonstrates how to use Appyx within Google's Jetpack Compose Navigation library.
+
+This example may be useful if you need to migrate to Appyx gradually.
```
Comment (zsoltk) [inline samples/navigation-compose/README.md:5]: ?

```suggestion
This example may be useful if:

- You want to to use some `RoutingSources` in your non-Appyx app
- You want to migrate to Appyx gradually
```

**Review by zsoltk — APPROVED** [review]

File: samples/navigation-compose/README.md (hunk 2 above)
Comment (LachlanMcKee) [reply samples/navigation-compose/README.md:5]: done

File: documentation/how-to-use-appyx/sample-apps.md (hunk 1 above)
Comment (LachlanMcKee) [reply documentation/how-to-use-appyx/sample-apps.md:18]: done

**Pull request merged**